policykit/policyengine/models.py

```diff
@@ -760,10 +760,6 @@ def save(self, *args, **kwargs):
 
 class PlatformActionBundle(BaseAction):
 
-    bundled_actions = models.ManyToManyField(PlatformAction)
```

Comment:
again, why are we deleting this field?

Reply:
This line already exists in the file in line 801 (so this line is a repeat).

Reply:
yikes o_o